### PR TITLE
Enable gpb build on Travis CI

### PR DESCRIPTION
**Pull request by lukebakken**
The purpose is to add continuous integration builds to `gpb`

### CODE REVIEW

**Comment by lukebakken** [comment]
@tomas-abrahamsson this is all set.

To enable Travis CI for your `tomas-abrahamsson/gpb` repository, visit this page:

https://travis-ci.org/profile/tomas-abrahamsson

You should then be able to "flip the switch" for your `gpb` repository. Then, you can click the gear next to the repository name to change some settings. This is how I have `lukebakken/gpb` set up:

travis-ci-gpb-settings

**Comment by tomas-abrahamsson** [comment]
Wow, I've been meaning to do this for a long time, thanks a bunch for preparations!

**Comment by lukebakken** [comment]
Glad to help! Let me know if you have any issues starting the build in your `tomas-abrahamsson` organization on Travis CI

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      - libprotobuf-dev
+      - protobuf-compiler
+otp_release:
+  - 19.1
+  - 18.3
+  - 17.5
+  - R16B03
+  - R15B03
+env:
+  - VERBOSE=1
```
Comment (lukebakken) [inline .travis.yml]: This isn't strictly necessary, but Travis will time out after 10 minutes if no output is echoed and `VERBOSE` output will prevent that.

Comment (tomas-abrahamsson) [inline .travis.yml]: Ah, I see :)

**Comment by tomas-abrahamsson** [comment]
I Just pushed 3.26.2 including this.

**Comment by lukebakken** [comment]
Awesome, thanks!

**Comment by tomas-abrahamsson** [comment]
I noticed you had upped the timeout of the `translation_of_google.protobuf.Any_test`. In my local Jenkins, it seems to take at most about 0.1 to 0.15 second, and also in your build history too (didn't check all). Many of the tests in the `gpb_compile_tests` take around 0.1 to 0.2 seconds. I guess it is only natural that the jitter on Travis CI can be expected to be much higher than when running locally, so I'll keep an eye out for if this will have to be adjusted for more tests.

Nevertheless, a big thanks for this! It's a big step forward

**Comment by lukebakken** [comment]
That particular test didn't time out on my local workstation, but, interestingly enough, `never_generates_unused_translator_functions_test` did start timing out consistently on my workstation and Travis. Like you said due to jitter on Travis I wouldn't be surprised if other tests become a little less dependable.

In the future, it is possible to only increase the timeout if the `TRAVIS` env variable is present, if necessary.

**Pull request merged**